BRANDING.MD

```diff
@@ -1,37 +1,49 @@
-## Knative Branding Guidelines
-
-The Knative logo and trademark are made available here with the following
-guidelines:
-
-**You may:**
- - use them for non-commercial uses such as t-shirts and stickers
- - use them to link to knative.dev
- - use them in a blog post or news article about Knative
- - use them to advertise that your product has support for Knative [Note that
-   at this time such usage does *not* indicate conformance with any standards
-   or requirements]
-
-**You may not:**
- - Use them in a way that would confuse people about the origin of Knative,
-   its maintainers or governance structure
- - Use them as your product's logo or trademark
- - Create a modified version of them (including, but not limited to, changing
-   of colors, shape or scale)
- - Integrate them into your logo
- - Use them in such a manner as to create the impression that your product or
-   service has been certified, vetted, or otherwise endorsed by the Knative
-   project
-
-If the above guidelines do not cover your potential use case, or you're just
-not sure; please file an [issue](https://github.com/knative/community/issues)
-for the Trademark Committee to review. Generally, the Trademark Committee is
-likely to approve respectful variations of the Knative logo for non-commercial
-physical items ("swag"), but would like to review the usage first.
-
-The Knative Trademark Committee understands the community desire for a
-conformance standard, and we do plan to implement one at a future date. We
-also recognize that conformance is a complicated issue with many variables,
-and it is still early days for the Knative project. For comparison, the
-Kubernetes project did not implement the [Certified Kubernetes Conformance
-Program](https://github.com/cncf/k8s-conformance) until version 1.7. Our goal
-is to have a Conformance Working Group created soon.
+## Knative Brand Guidelines
+
+The Knative trademark and ![Knative Logo](icons/logo.svg "The Knative Logo")
```

Comment:
This seems to conflict with:
https://github.com/knative/community/pull/544
@duglin @aijamalnk

Reply:
Do mean the reference to google? If so, I think it's ok since the other PR talks about "overseeing" while this talks about "ownership".  But it might be nice to have similar language so maybe we can steal the wording from #544 ?

Reply:
Yes, that as well as the fact that depending on the order we merge these, there will be merge conflicts?